### PR TITLE
Bump 2.28.2

### PR DESCRIPTION
**Pull request by matthewfala**
*Issue #, if available:*
Bump to 2.28.2, also resolve load test tcp logger security issue

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
 # Changelog
+### 2.28.2
+This release includes:
+* Fluent Bit [1.9.9](https://fluentbit.io/announcements/v1.9.9/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.0
+* Amazon Kinesis Streams for Fluent Bit 1.10.0
+* Amazon Kinesis Firehose for Fluent Bit 1.7.0
+
+Compared to `2.28.1` this release adds the following feature that we are working on getting accepted upstream:
+* Bug - Stop trace_error from truncating the OpenSearch API call response [fluentbit:5788](https://github.com/fluent/fluent-bit/pull/5788)
```
Comment (PettitWesley) [inline CHANGELOG.md:10]: wait I don't see this in our cherry picks??

Comment (PettitWesley) [reply CHANGELOG.md:10]: this was merged upstream?

Comment (PettitWesley) [reply CHANGELOG.md:10]: I think this is a just a fix in the upstream release?

File: AWS_FLB_CHERRY_PICKS [hunk 2]
```diff
@@ -1,2 +0,0 @@
-# Resolve long tag segfault issue. Without this patch, Fluent Bit may segfault if it encounters tags over 256 characters in length. [fluentbit:5753](https://github.com/fluent/fluent-bit/issues/5753). Patch commit - https://github.com/matthewfala/fluent-bit/commits/immutable-1.9-long-tag
-https://github.com/matthewfala/fluent-bit immutable-1.9-long-tag a74f095d0974d13e7a0539f7721f0b6a4e6b1e63
```
Comment (matthewfala) [inline AWS_FLB_CHERRY_PICKS:2]: actually this was merged into upstream.

File: CHANGELOG.md (hunk 1 above)
Comment (matthewfala) [reply CHANGELOG.md:10]: Oh actually the cherry pick can be removed. This is an actual change that was merged in.

**Review by PettitWesley — APPROVED** [review]

**Pull request merged**